README.md

```diff
@@ -1,6 +1,6 @@
-<!-- markdownlint-disable-next-line -->
+<!-- markdownlint-disable-file -->
```

Comment:
Does this disable linting for the whole file? 
Instead could you just disable linting for the specific lines that are the issue?